src/index.ts

```diff
@@ -164,6 +169,28 @@ router.get('/exchangeRate', async function(req, res) {
     }
   } while (inversePair && response == null)
 
+  // Use fallback hardcoded rates if lookups failed
+  if (
+    response == null &&
+    fallbackConstantRatePairs[`${currencyA}_${currencyB}`] != null
+  ) {
+    response = {
+      rate: fallbackConstantRatePairs[`${currencyA}_${currencyB}`],
+      needsWrite: false
+    }
+  }
+
+  if (
+    response == null &&
+    fallbackConstantRatePairs[`${currencyB}_${currencyA}`] != null
+  ) {
+    response = {
+      rate: fallbackConstantRatePairs[`${currencyB}_${currencyA}`],
+      needsWrite: false
```

Comment:
Needswrite should be true so that we write this to the database and not look up all over again each time